### PR TITLE
Working flow training->prediction

### PR DESCRIPTION
**Pull request by dordor7**
.erl changes where requested by Tal, and must be approved by him!

### CODE REVIEW

File: src_erl/Communication_Layer/http_Nerlserver/src/nerlNetServer_app.erl [hunk 1]
```diff
@@ -45,17 +45,17 @@
...
 
     %Create a listener that waits for a message from python about the adresses of the wanted json
-    %createNerlnetInitiator(HostName),
+    createNerlnetInitiator(HostName),
     receive 
 
         {jsonAddress,MSG} -> {ArchitectureAdderess,CommunicationMapAdderess} = MSG
         %%TODO remove this "after" part when python is ready to send jsonPaths
-        after 10000 ->   JsonPath = "../../../jsonPath",
+        after 1000000 ->   JsonPath = "../../../jsonPath",
```
Comment (leondavi) [inline src_erl/Communication_Layer/http_Nerlserver/src/nerlNetServer_app.erl:58]: Add comment - what is the purpose of this PR
better to use macro (?TIMEOUT_SOMETHING) 

@kapelnik

File: src_py/apiServer/transmitter.py [hunk 2]
```diff
@@ -96,15 +100,16 @@ def predict(self):
...
 
         while globe.pendingAcks > 0:
             time.sleep(0.005)
             pass 
 
         # 1 Ack for startCasting():
         globe.pendingAcks += 1
-        self.startCasting()
+
+        self.startCasting(10)
```
Comment (leondavi) [inline src_py/apiServer/transmitter.py:112]: What does it means startCasting(10) ? why 10 and not 15? 
Please use macro like or add a short comment

File: inputJsonFiles/experimentsFlow/expFlowJson.json [hunk 3]
```diff
@@ -3,23 +3,23 @@
      {
         "source name": "s1",
         "workers": "w1",
-        "CSV path": "HeartData/heart2020_train"
+        "CSV path": "OneTwo/OneTwo_train"
```
Comment (leondavi) [inline inputJsonFiles/experimentsFlow/expFlowJson.json:6]: Can we use several expFlowJson? 
Did you check the Jupyter API that it works?

File: jsonPath [hunk 4]
```diff
@@ -1,7 +1,7 @@
-../../../inputJsonFiles/Architecture/jsonArch1PC1Worker.json
-../../../inputJsonFiles/ConnectionMap/1PC1Worker.json
+/usr/local/lib/nerlnet-lib/NErlNet/inputJsonFiles/Architecture/jsonArch1PC1Worker.json
```
Comment (leondavi) [inline jsonPath:1]: Great!

**Review by kapelnik — APPROVED** [review]

**Review by leondavi — APPROVED** [review]

**Pull request merged**